packages/pipeline/package.json

```diff
@@ -7,10 +7,13 @@
         "build": "yarn tsc -b",
         "build:ci": "yarn build",
         "test": "yarn run_mocha",
-        "rebuild_and_test": "run-s build test",
+        "rebuild_and_test": "run-s build test:all",
+        "test:db": "yarn run_mocha:db",
+        "test:all": "run-s test test:db",
         "test:circleci": "yarn test:coverage",
-        "run_mocha": "mocha --require source-map-support/register --require make-promises-safe lib/test/**/*_test.js --bail --exit",
-        "test:coverage": "nyc npm run test --all && yarn coverage:report:lcov",
+        "run_mocha": "mocha --require source-map-support/register --require make-promises-safe 'lib/test/!(entities)/**/*_test.js' --bail --exit",
+        "run_mocha:db": "mocha --require source-map-support/register --require make-promises-safe lib/test/db_global_hooks.js 'lib/test/entities/*_test.js' --bail --exit --timeout 60000",
```

Comment:
I've never used root-level hooks before and did a quick search for docs. Should we add the `--delay` flag to mocha like described [here](https://mochajs.org/#delayed-root-suite)?

Reply:
If we need any delays, I would prefer to add them in TypeScript so that they are both more visible and more flexible.